tests/python/test_fields_builder.py

```diff
@@ -3,174 +3,276 @@
 
 import taichi as ti
 
+'''
+Test fields with shape.
+'''
+
 
 @ti.test(arch=[ti.cpu, ti.cuda, ti.vulkan, ti.metal])
 def test_fields_with_shape():
-    n = 5
-    x = ti.field(ti.f32, [n])
 
+    # Initialize common variables for tests below
+    shape_size_1d = 5
+    x = ti.field(ti.f32, [shape_size_1d])
+    y = ti.field(ti.f32, [shape_size_1d])
```

Comment:
```suggestion
    x = ti.field(ti.f32, shape=shape_size_1d)
    y = ti.field(ti.f32, shape=shape_size_1d)
```
This is more common practice in Taichi.

Reply:
OK